### ISSUE TITLE
Running projects in sequential order, files in parallel

### ISSUE THREAD

**Issue by FrankNutreco** [microsoft/playwright#19889]
I have found the following PR which addresses my issue:
https://github.com/microsoft/playwright/pull/16698

This is exactly what I need. However, the PR is closed with mention of a new API proposal. I cannot seem to find this new API proposal, and I very much would like to use this functionality.

![image](https://user-images.githubusercontent.com/107055245/210761065-dc5ad7b4-416b-4c36-8788-7e96f744ef9b.png)

**Comment by mxschmitt** [comment]
Could you elaborate what you need? This would help a ton. Most likely something like https://github.com/microsoft/playwright/issues/14084 or https://github.com/microsoft/playwright/issues/17553.

Then we can fold it into them.

**Comment by FrankNutreco** [comment]
Sure! Thanks for the reply.

I would like to group files and run them in parallel. When this group completes, have the next defined group of files run in parallel.

For example:
Group 1
File A, file B
When all the files in group 1 are done running in parallel, run:
Group 2
File C
When all the files in group 2 are done running in parallel, run:
Group 3
File D, file E
etc.

So file A and file B are run at the same time
Then file C
Then file D and file E

**Comment by FrankNutreco** [comment]
It looks like your link https://github.com/microsoft/playwright/issues/17553 which refers to https://github.com/microsoft/playwright/pull/17548 contains the solution. 

However, I cannot find any documentation on how to set this up in config. Could you point me in the right direction or give a (brief) example on how this would be setup? @mxschmitt

**Comment by mxschmitt** [comment]
This stuff is not released yet, thats why https://github.com/microsoft/playwright/issues/17553 is still open.

**Comment by FrankNutreco** [comment]
Too bad, but thanks for looking into it.

**Comment by Kamal-Yalikun** [comment]
Our scenario is a little differenet:

We would like to group files and run only the groups in parallel. 

For example:
Group 1
File A, file B

Group 2
File C, File D

Group 3
File E, file F
etc.

Group1, Group2 and Group3 Should be run in parallel, but the spec files in each group has to run in sequential order

**Comment by buehlerfa** [comment]
Any update on this feature?  Was it ever released/ are there any plans to release something like #17548?

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you have additional information not present in this issue that you think will help prioritizing it, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
